### ISSUE TITLE
Interesting Cross-Reference

### ISSUE THREAD

**Issue by wingman-jr-addon** [iridiumblue/roc-star#6]
I landed here because I previously used Tensorflow research's "global objectives" ROC AUC loss and was trying to help someone else with interest find a PyTorch equivalent. You can see the removed TF code here: https://github.com/tensorflow/models/tree/3e73c76c7a5373dafd71ef9231896dabcb696cc5/research/global_objectives
I was trying to see if the original authors could add a note about how to do the upgrade in the issue https://github.com/tensorflow/models/issues/8062 but no dice.

What's interesting is that it appears they based their objective on another paper, [Scalable Learning of Non-Decomposable Objectives](https://arxiv.org/abs/1608.04802). It looks like the paper is newer, but I'd be curious to see how the author here thinks the approach stacks up from a theoretical perspective. What do you think?
